### PR TITLE
fix bunx-bins verdaccio package

### PR DESCRIPTION
**Pull request by paperclover**
### What does this PR do?

1. remove the python test since not all machines have `python3` (it is not in our devenv setup guide)
2. compile native.exe for baseline cpu. this will fix install registry in windows

### CODE REVIEW

**Comment by nektro** [comment]
`windows bin linking shim should work` test is still failing

File: test/cli/install/registry/packages/bunx-bins/package.json [hunk 1]
```diff
@@ -43,7 +44,7 @@
 	"_distfiles": {},
 	"_attachments": {
 		"bunx-bins-1.0.0.tgz": {
-			"shasum": "c23382ce43a04a94035050ac736f04179a8639ff",
+			"shasum": "a77e200b47158585a4bf70ef934202c5fb2b0dd7",
 			"version": "1.0.0"
 		}
```
Comment (jdalton) [inline test/cli/install/registry/packages/bunx-bins/package.json:49]: ☝️ Is this produced from a pack'd npm/bun package? Are there documented steps with how to update this test when needed?

**Review by jdalton — APPROVED** [review]

**Review by nektro — CHANGES_REQUESTED** [review]
see comment above, relevant test is still not passing as of writing

File: test/cli/install/registry/packages/bunx-bins/package.json (hunk 1 above)
Comment (paperclover) [reply test/cli/install/registry/packages/bunx-bins/package.json:49]: yes, it can be updated using `npm publish`

**Pull request merged**